### DISCUSSION TITLE
Welcome to swig Discussions!

### DISCUSSION THREAD

**Discussion by wsfulton** [swig/swig#1906] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vadz** [comment]
Are Discussions going to replace or supplement the mailing list?

**Comment by wsfulton** [comment]
I've never used these discussions before, but thought we'd try it out as a supplement to mailing lists. Maybe replace the mailing lists if it goes well??

**Reply by ojwb** [reply]
It doesn't seem to be going well.

Very few users are using it, but even fewer people are helping the ones who do ask questions here.  Almost all of the activity lately has been me trying to clean up.  I think we should conclude it's a failed experiment and shut it down, directing questions to swig-user instead.  They won't necessarily get answered there either, but a lot more people seem to read that and I think users stand much more chance there than here.
